plugins/follow_me/follow_me.h

```diff
@@ -107,7 +107,7 @@ class FollowMe
     bool is_active() const;
 
     /**
-     * @brief Geographical location of the target in motion.
+     * @brief Type for Geographical location of the target in motion.
```

Comment:
probably better as "Type for geographical location of the target being followed." - the target may not be in motion :-)

Reply:
Thanks @hamishwillee . I'll correct.